samples/event_manager/sample.yaml

```diff
@@ -1,12 +1,24 @@
 sample:
   description: Sample showing Event Manager usage
   name: Event Manager sample
+common:
+  tags: introduction
+  harness: console
+  integration_platforms:
+    - native_posix
+    - nrf52dk_nrf52832
+    - nrf52840dk_nrf52840
+    - nrf9160dk_nrf9160_ns
+    - nrf21540dk_nrf52840
+  harness_config:
+    type: multi_line
+    ordered: false
```

Comment:
Well, actually it may be a little bit misleading. We can't assure order in a way we would like it be, because events are submitted repeatedly. For example if we search for ordered sequence: "dog", "cat", "pet" and our logs will be "pet", "cat", "dog", "pet", "cat", "dog", "pet", "cat", "dog" our test will also pass. Regarding this, I would opt to keep `ordered: False`.

Reply:
let's keep it as is at the moment